### DISCUSSION TITLE
ExpressJS works inside nodejs-mobile?

### DISCUSSION THREAD

**Discussion by goodwill** [nodejs-mobile/nodejs-mobile#89] [Q&A] [answered]
As above, can I use expressJS to serve a website inside my app and consume the website in the same app's WebView?

**Comment by staltz** [comment, marked as answer]
In theory, yes it should work. I haven't tested specifically `express` so I can't testify that it isn't without quirks or obstacles. But I can testify that you can run an HTTP server from nodejs-mobile such that a webview (or even browser connecting to localhost) can consume it.
